src/components/customMdx/code.tsx

```diff
@@ -245,7 +245,9 @@ const LineNo = styled.span`
   line-height: ${p => p.theme.space[24]};
   color: ${p => p.theme.colors.gray400};
   display: inline-block;
-  text-align: right; // padding-left: 1em; user-select: none; width: 24px;
+  text-align: right;
+  user-select: none; 
+  width: 24px;
```

Comment:
```suggestion
  width: 30px;
```

As some of code may go beyond **100+** lines codes,  but might not go **1000+** I  believe
So if 3 digit letter comes in, then, it looks bit odd

![layout](https://user-images.githubusercontent.com/24385409/102963673-b6e1a800-450f-11eb-9a1a-1f95e7b5e302.jpg)

So batter go with atleast `30px`